docs/crashlytics/usage.mdx

```diff
@@ -1,6 +1,171 @@
 ---
-title: Crashlytics
+title: Using Firebase Crashlytics
 sidebar_label: Usage
 ---
 
-Crashlytics usage
+To start using Firebase Crashlytics within your project, import it at the top of your project files:
+
+```dart
+import 'package:firebase_crashlytics/firebase_crashlytics.dart';
+```
+
+## Sending reports to Crashlytics
```

Comment:
nit: may want to choose being consistent on using title case (e.g. see reports.mdx) vs sentence casing